includes/wc-order-functions.php

```diff
@@ -719,19 +719,30 @@ function wc_processing_order_count() {
  * @return int
  */
 function wc_orders_count( $status ) {
-	$count = 0;
+	global $wpdb;
 
+	$count = 0;
+	$status = 'wc-' . $status;
 	$order_statuses = array_keys( wc_get_order_statuses() );
 
-	if ( ! in_array( 'wc-' . $status, $order_statuses ) ) {
+	if ( ! in_array( $status, $order_statuses ) ) {
 		return 0;
 	}
 
+	$cache_key = "wc-orders-{$status}";
```

Comment:
Please, change it to:

```php
$cache_key = WC_Cache_Helper::get_cache_prefix( 'orders' ) . $status;
```

So this make easy to invalidate the cache when running functions like `wc_delete_shop_order_transients()`.